### PR TITLE
TW-252: verify wording for TW-246

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
This PR is meant to verify the information changed on the limits page.

## Notes to reviewers
Needs approval from @yiminc and/or @mfateev . 

There are other wordage changes as well. This is really not a great example of a KB article, but for now, I just want to get these numbers and phrases right.

### CODE REVIEW

File: kb/temporal-platform-limits-sheet.md [hunk 1]
```diff
@@ -8,18 +8,17 @@ tags:
...
 
-Running into limits can cause unexpected failures, so be mindful when you design your systems.
+Running into limits can cause unexpected failures.
+Knowing Temporal's limits can prevent that.
 
-Here is a list of many hard (error) or soft (warn) limits that you could encounter while using the Temporal Platform.
-
-Unless otherwise stated, Event limits apply to all Events occurring simultaneously on a system.
+This page details many of the hard and soft limits that are coded into the Temporal Platform.
+Hard limits will fail with an error; soft limits will warn the user about the limit to be exceeded.
```
Comment (yiminc) [inline kb/temporal-platform-limits-sheet.md:15]: Currently, exceeding a soft limit will produce a warning log on server side.

File: kb/temporal-platform-limits-sheet.md [hunk 2]
```diff
@@ -37,4 +36,5 @@ Unless otherwise stated, Event limits apply to all Events occurring simultaneous
     - `SignalExternalWorkflowExecution`
```
Comment (yiminc) [inline kb/temporal-platform-limits-sheet.md:36]: This should be in its own section (concurrent action limit), and not be part of History total count limit.
Also, I would not call them as Events, but rather Actions.

**Comment by DJSanti** [comment]
@yiminc see recent push for changes. Trying to figure out how much more to edit.

File: kb/temporal-platform-limits-sheet.md [hunk 3]
```diff
@@ -32,9 +31,10 @@ Unless otherwise stated, Event limits apply to all Events occurring simultaneous
...
-  - We fail the following Commands at 50,000 Events for OSS and 2,000 Events for Cloud:
-    - `ScheduleActivityTask`
-    - `SignalExternalWorkflowExecution`
-    - `RequestCancelExternalWorkflowExecution`
-    - `StartChildWorkflowExecution`
+- We fail the following Commands at 50,000 Events for OSS and 2,000 Events for Cloud:
+  - `ScheduleActivityTask`
+  - `SignalExternalWorkflowExecution`
+  - `RequestCancelExternalWorkflowExecution`
+  - `StartChildWorkflowExecution`
```
Comment (flossypurse) [inline kb/temporal-platform-limits-sheet.md:38]: I think what @yiminc is saying is that we fail the following Commands at 2,000 **concurrent** Actions (note this NOT the same as Events) ?

**Review by flossypurse — CHANGES_REQUESTED** [review]
I think the details are still off.

File: kb/temporal-platform-limits-sheet.md (hunk 3 above)
Comment (DJSanti) [reply kb/temporal-platform-limits-sheet.md:38]: okay I gotcha

**Comment by DJSanti** [comment]
@flossypurse That's fair. I could still use some input from @yiminc for this.

File: kb/temporal-platform-limits-sheet.md [hunk 4]
```diff
@@ -32,7 +31,8 @@ Unless otherwise stated, Event limits apply to all Events occurring simultaneous
   - We warn at 10,000 Events: `history size exceeds warn limit.`
   - We error at 50,000 Events: [`history size exceeds error limit.`](https://github.com/temporalio/temporal/blob/v1.7.0/service/history/workflowExecutionContext.go#L1204)
   - This is configurable with [`HistoryCountLimitError` and `HistoryCountLimitWarn`](https://github.com/temporalio/temporal/blob/v1.7.0/service/history/configs/config.go#L382-L383), if you know what you are doing.
-  - We fail the following Commands at 50,000 Events for OSS and 2,000 Events for Cloud:
+- **Concurrent Action limit**
+  - We fail the following Commands at 50,000 Events for OSS and 2,000 concurrent Actions for Cloud:
```
Comment (yiminc) [inline kb/temporal-platform-limits-sheet.md:35]: We fail the following action Commands if concurrent running action count exceeds limit. The limit on cloud is 2000. There is no default limit on OSS server, but can ben enforced by dynamic config...

**Review by yiminc — APPROVED** [review]

File: kb/temporal-platform-limits-sheet.md [hunk 5]
```diff
@@ -8,18 +8,17 @@ tags:
...
 
-Running into limits can cause unexpected failures, so be mindful when you design your systems.
+Running into limits can cause unexpected failures.
+Knowing Temporal's limits can prevent that.
 
-Here is a list of many hard (error) or soft (warn) limits that you could encounter while using the Temporal Platform.
-
-Unless otherwise stated, Event limits apply to all Events occurring simultaneously on a system.
+This page details many of the hard and soft limits that are coded into the Temporal Platform.
+Hard limits will fail with an error; soft limits will produce a warning log on the server side.
```
Comment (flossypurse) [inline kb/temporal-platform-limits-sheet.md:15]:
```suggestion
Hard limits fail with an error; soft limits produce a warning log on the server side.
```

**Review by flossypurse — APPROVED** [review]

**Comment by DJSanti** [comment]
Added dynamic value config via @aarohib . Need another check from @yiminc and @flossypurse

**Comment by DJSanti** [comment]
@flossypurse @yiminc Passed a minor change as to how the variables should be written. Would appreciate another pass.

**Review by yiminc — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Pull request merged**